waterlog/src/Components/SensorsTable.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import {  fetchSensors } from '../actions/SensorsActions';
+import { formatDate, getStatusIcon } from './../utils'
+import Loader from './Loader';
+
+class SensorTableComponent extends Component {	
+	componentDidMount() {
```

Comment:
Your indentation is all over the place please be consistent. 
Here you used 2 `waterlog/src/Components/DailyTankGraph.js`
and in this file you used clearly 4 tabs

Reply:
Please look at all the files you committed

Reply:
I've changed all the files  to be 2 tabs